apps/frontend/src/App.tsx

```diff
@@ -2,21 +2,21 @@ import React from 'react';
 import { Router, Switch, Route } from 'react-router-dom';
 
 import Admin from './admin/Admin';
-import ContinueOnboarding from './onboarding/ContinueOnboarding';
 import CreateProfile from './onboarding/CreateProfile';
 import history from './history';
 import Home from './home/Home';
 import News from './news/Routes';
 import Teams from './teams/Routes';
 import Users from './users/Routes';
-import Welcome from './onboarding/Welcome';
+import Welcome from './welcome/Routes';
 
 import { AuthProvider } from './auth';
+import CheckAuth from './auth/CheckAuth';
 
 const AuthCallbackGuardedHome: React.FC<{}> = () => (
-  <ContinueOnboarding>
+  <CheckAuth>
```

Comment:
Securing even the home page now that we show guard with a login makes sense, but don't we still want `ContinueOnboarding` around that? Otherwise the onboarding flow will break by not triggering the connect-by-code and then profile creation

Reply:
We can bring the Onboarding when it's better defined.

Reply:
But this *breaks* the functionality. Connecting user accounts will not be possible. I wouldn't be able to even test the ticket that I just completed :/